apps/wallet/src/ui/app/redux/store/index.ts

```diff
@@ -14,7 +13,7 @@ const store = configureStore({
 			thunk: {
 				extraArgument: thunkExtras,
 			},
-		}).prepend(amplitudePersistenceMiddleware.middleware),
```

Comment:
was this intentional?

Reply:
Yes I removed the accounts slice and this is not used anywhere else. I added the functionality to enable events in App component [here](https://github.com/MystenLabs/sui/pull/13371/files/567ed836f103d4eb4591b4ce78999e83612339b5#diff-649a9e9eccd1521dbf1f43dc84ab74b13ad124f2be9f96208d94e9539d1cd0abR90) cc @williamrobertson13